docker-compose/.env_ucentralgw-ui

```diff
@@ -0,0 +1,2 @@
+DEFAULT_GATEWAY_URL=https://ucentral.dpaas.arilia.com:16001
```

Comment:
Why do we point to this gateway and not the one we are deploying?

Reply:
Left this value because the hostname of the gateway is specific to every deployment and I can't reference any variable here.

Reply:
Understand but the Docker compose deployment contains a gateway that we could point to, doesn't it?

Reply:
You mean by fixed Docker Compose service name `ucentralgw`? I think that doesn't work since the specified gateway URL is requested on the client side.

Reply:
@brennerm does that answer your question or do we talk cross purposes?

Reply:
Agree but the gateway is also exposed on localhost according to this configuration, right?
```
      - "127.0.0.1:15002:15002"
      - "127.0.0.1:16001:16001"
      - "127.0.0.1:16003:16003"
   ```

Reply:
Yes, but configuring localhost as the gateway URL will throw SSL errors when doing XHR requests to the REST API.
I'll leave this value for now and will change it when I add the generic wildcard certificates for local dev usage to the repository.